src/Channel/__tests__/Home.spec.jsx

```diff
@@ -0,0 +1,133 @@
+import { render, screen, fireEvent, waitFor } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import axios from "axios";
+import { MemoryRouter } from "react-router-dom";
+import { describe, test, expect, beforeEach, vi } from "vitest";
+
+import Home from "@/Channel/components/Home";
+import useCategorizeChannels from "@/Channel/hooks/useCategorizeChannels";
+
+vi.mock("axios");
+vi.mock("@/Channel/hooks/useChannels", () => ({
+  default: vi.fn(),
+}));
+vi.mock("@/Channel/hooks/useCategorizeChannels", () => {
+  const mockedCategorize = vi.fn();
+  return {
+    default: mockedCategorize,
+    __esModule: true,
+  };
+});
+vi.mock("@/User/hooks/useUserData", () => ({
+  default: vi.fn(),
+}));
+vi.mock("@/User/hooks/useUserId", () => ({
+  default: vi.fn(),
+}));
+vi.mock("@/Channel/hooks/useChannelNavigation", () => ({
+  default: () => mockNavigate,
+}));
+vi.mock("@/Channel/hooks/useToggleFavorite", () => ({
+  default: () => mockToggleFavorite,
+}));
+
+const mockNavigate = vi.fn();
+const mockToggleFavorite = vi.fn();
+
+describe("Home 컴포넌트 통합 테스트", async () => {
+  beforeEach(() => {
+    vi.clearAllMocks();
+    axios.get = vi.fn();
+    vi.mocked(useCategorizeChannels).mockReturnValue([[], []]);
+  });
+
+  test("라디오 목록의 채널을 클릭하면 해당 라디오 채널로 이동한다", async () => {
+    const mockChannelData = [
+      { id: 1, name: "KBS Radio", logoUrl: "kbs.jpg" },
+      { id: 2, name: "MBC Radio", logoUrl: "mbc.jpg" },
+    ];
+    vi.mocked(useCategorizeChannels).mockReturnValue([[], mockChannelData]);
+
+    render(
+      <MemoryRouter>
+        <Home />
+      </MemoryRouter>
+    );
```

Comment:
```suggestion
  const renderHome = () =>
    render(
      <MemoryRouter>
        <Home />
      </MemoryRouter>
    );
```
테스트 코드를 잘 짜주신 거 같아요~주요 동작 흐름이 눈에 잘 보입니다😊

그렇지만 `render(<MemoryRouter><Home /></MemoryRouter>)` 코드 부분은 테스트 코드 내에서 동일하게 반복되고 있어서, `renderHome()` 같은 함수로 분리하면 중복이 줄어들고 코드 가독성도 더 좋아질 것 같아요!

Reply:
좋은 리뷰 감사드립니다.

말씀하신대로 `renderHome`이라는 함수를 만들어 호출하기만 해도 동일하게 동작하네요! 해당 사항 반영하겠습니다~